doc/Language/objects.pod

```diff
@@ -65,7 +65,7 @@ return value of a method call. This is how read-writable attributes to
 objects are used:
 
 =for code :allow<L>
-$*IN.L<nl-in> = "\r\n";
+$*IN.C<nl-in> = "\r\n";
```

Comment:
@zoffixznet Absolutely.

I'm not sure why it has to be changed though. ``nl-in`` is not documented yet (https://github.com/perl6/doc/issues/484), but it does not mean that we should not have a link like this.